dl/runner.py

```diff
@@ -9,13 +9,13 @@
 
 from catalyst.utils.factory import UtilsFactory
 from catalyst.utils.misc import merge_dicts
-from catalyst.dl.callbacks import Callback, CheckpointCallback, InferCallback
+from catalyst.dl.callbacks.core import Callback, CheckpointCallback, InferCallback
```

Comment:
is it possible to import all callbacks just from `catalyst.dl.callbacks`? maybe, add them to __init__?